src/web/app/components/session-edit-form/session-edit-form.component.ts

```diff
@@ -198,6 +201,15 @@ export class SessionEditFormComponent implements OnInit {
     }
   }
 
+  /**
+   * Handles cancel button click event by updating instructions
```

Comment:
`by updating instructions`? What does it mean?

Reply:
I honestly don't recall. I'll update to just `Handles cancel button click event.`